tests/e2e/test_raster_analysis.py

```diff
@@ -0,0 +1,3811 @@
+                (102.09083557, -0.28053856),
+                (102.08932495, -0.28156203),
+                (102.08332062, -0.28568479),
+                (102.08195496, -0.28662714),
+                (102.08029938, -0.28771943),
+                (102.079216, -0.28845474),
+                (102.07881927, -0.28872541),
+                (102.07398987, -0.29203692),
+                (102.07363129, -0.29229203),
+                (102.07197571, -0.29342717),
+                (102.07027435, -0.29459229),
+                (102.0670929, -0.29680428),
+                (102.06536102, -0.29800752),
+                (102.06391144, -0.29911157),
+                (102.05745697, -0.30402023),
+                (102.05377197, -0.30694413),
+                (102.05212402, -0.30827996),
+                (102.04995728, -0.30999333),
+                (102.04745483, -0.31200412),
+                (102.04576111, -0.31366625),
+                (102.0457077, -0.31371894),
+                (102.04217529, -0.31733423),
+                (102.04091644, -0.31896323),
+                (102.04045105, -0.3195661),
+                (102.04026794, -0.31982914),
+                (102.03816223, -0.32261443),
+                (102.03749847, -0.32337788),
+                (102.03709412, -0.32382923),
+                (102.03570557, -0.32538024),
+                (102.03457642, -0.32668656),
+                (102.03281403, -0.32852831),
+                (102.03227997, -0.3290998),
+                (102.0316391, -0.32981434),
+                (102.03051758, -0.33193311),
+                (102.03029633, -0.33241495),
+                (102.02961731, -0.33382416),
+                (102.02933502, -0.33440635),
+                (102.02857208, -0.33572114),
+                (102.02561951, -0.34032384),
+                (102.02493286, -0.34139612),
+                (102.01876831, -0.35006911),
+                (102.01642609, -0.35257882),
+                (102.0162735, -0.35274804),
+                (102.01199341, -0.35740611),
+                (102.00775146, -0.36193576),
+                (102.00698853, -0.36267963),
+                (102.00512695, -0.36455712),
+                (102.00331116, -0.36636376),
+                (101.99982452, -0.36973611),
+                (101.99729156, -0.37223095),
+                (101.99599457, -0.37350416),
+                (101.9943161, -0.37512147),
+                (101.99274445, -0.37670442),
+                (101.991539, -0.37784886),
+                (101.99123383, -0.37813246),
+                (101.99000549, -0.37926799),
+                (101.98730469, -0.38195565),
+                (101.98697662, -0.38225827),
+                (101.98672485, -0.38249689),
+                (101.9859314, -0.38316917),
+                (101.98496246, -0.38400757),
+                (101.98412323, -0.38473919),
+                (101.98397064, -0.38490552),
+                (101.98194122, -0.38713297),
+            )
+        ],
+    ],
+}
```

Comment:
Please move this into a seperate file in a fixture folder to make test more readable